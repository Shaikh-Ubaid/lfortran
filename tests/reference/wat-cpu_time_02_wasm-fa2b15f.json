{
    "basename": "wat-cpu_time_02_wasm-fa2b15f",
    "cmd": "lfortran --no-color --show-wat {infile}",
    "infile": "tests/../integration_tests/cpu_time_02_wasm.f90",
    "infile_hash": "b059b12163601924d5ef98bed17705e08c0456055aa6c59be759794b",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "wat-cpu_time_02_wasm-fa2b15f.stdout",
    "stdout_hash": "ae5c49d66852993d568416c7b63f8742bd814206105d49bd90fe8b44",
    "stderr": "wat-cpu_time_02_wasm-fa2b15f.stderr",
    "stderr_hash": "eed305f72df00c3284e45e7a8e86ed7f291fc1d0aad1151e9b21f4c2",
    "returncode": 0
}