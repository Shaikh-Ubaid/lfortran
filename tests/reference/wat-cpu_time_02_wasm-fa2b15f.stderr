warning: WASM: C Intrinsic Functions not yet spported
 --> tests/../integration_tests/cpu_time_02_wasm.f90:6:5 - 9:19
  |
6 |        subroutine cpu_time(t) bind(c)
  |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^...
...
  |
9 |        end subroutine
  | ...^^^^^^^^^^^^^^^^^^^ cpu_time

warning: WASM: C Intrinsic Functions not yet spported
 --> tests/../integration_tests/cpu_time_02_wasm.f90:6:5 - 9:19
  |
6 |        subroutine cpu_time(t) bind(c)
  |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^...
...
  |
9 |        end subroutine
  | ...^^^^^^^^^^^^^^^^^^^ cpu_time
